### PR TITLE
Ap seen

### PR DESCRIPTION
**Pull request by bago2k4**
To test: https://docs.google.com/document/d/1oCo58bHM5pdc2otOIXuRaf_csj-VaNpS5qxSIK_05tQ/edit#

### CODE REVIEW

**Comment by belucid** [comment]
@bago2k4 merge conflicts here

**Comment by bago2k4** [comment]
@belucid conflicts solved

**Comment by belucid** [comment]
@bago2k4 merge conflict here again

**Comment by belucid** [comment]
@bago2k4 conflict here

**Comment by thepug** [comment]
I was able to recreate the issue but only the new posts are removed.

http://g.recordit.co/IMdFY4xEC0.gif

File: src/oc/web/actions/activity.cljs [hunk 1]
```diff
@@ -427,6 +427,83 @@
 
   (api/get-secure-activity (router/current-org-slug) (router/current-secure-activity-id) secure-activity-get-finish))
 
+;; Change reaction
+
+(defn activity-change [section-uuid activity-uuid]
+  (let [org-data (dis/org-data)
+        section-data (first (filter #(= (:uuid %) section-uuid) (:boards org-data)))
+        activity-key (dis/activity-key (:slug org-data) (:slug section-data) activity-uuid)
```
Comment (thepug) [inline src/oc/web/actions/activity.cljs]: can also use the `dispatcher/activity-data` function here

**Pull request merged**